guide/05-working-with-the-spatially-enabled-dataframe/part1_introduction_to_sedf.ipynb

```diff
@@ -0,0 +1,1555 @@
+{
```

Comment:
<p><em>"GeoSeriesAccessor</em> and <em>GeoAccessor</em> classes are similar to <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.html#pandas.Series" target="_blank">pandas.Series</a> and <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html#pandas.DataFrame" target="_blank">pandas.DataFrame</a> objects however they do not need to be called repeatedly during the analysis for usage.  </p><p>Importing Geo will add spatial and geom namespaces to your dataframe and series resp. It is best practice to import Pandas followed by importing both the <em>GeoAccessor</em> and the <em>GeoSeriesAccessor</em> classes at the beginning of the analysis. Importing these classes registers their functionality with Pandas and allows you to start performing spatial operations. You may import the classes as follows: "  ---&gt; </p><p> </p><p>"<em>GeoSeriesAccessor</em> and <em>GeoAccessor</em> classes are similar to <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.html#pandas.Series" target="_blank">pandas.Series</a> and <a href="https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.html#pandas.DataFrame" target="_blank">pandas.DataFrame</a> objects. However, you do not work with them directly. Instead, you import them right after you import Pandas as shown in the snippet below. Importing these classes registers the spatial functionality with Pandas and allows you to start performing spatial operations on your DataFrames.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>